core/error_list.h

```diff
@@ -87,8 +87,6 @@ enum Error {
 	ERR_HELP, ///< user requested help!!
 	ERR_BUG, ///< a bug in the software certainly happened, due to a double check failing or unexpected behavior.
 	ERR_PRINTER_ON_FIRE, /// the parallel port printer is engulfed in flames
-	ERR_OMFG_THIS_IS_VERY_VERY_BAD, ///< shit happens, has never been used, though
```

Comment:
Those are part of what makes the code fun to read IMO, alongside the previous `ERR_PRINTER_ON_FIRE`. If people vote for it, I'm not against removing it, but we lose a small part of our essence :P

Reply:
It could be renamed `ERR_OMG_THIS_IS_VERY_VERY_BAD` if the `F` is too offensive though.

Reply:
I'd drop the F if it has to be kept.

Reply:
They are replicated at three places (GDNative and C#) now and as part of the docs. I'd argue that is a bit much just for fun, and both of them contain "*fuck*", even if abbreviated. I'd prefer not to have them because of teaching etc., and just let "ERR_PRINTER_ON_FIRE" be so some fun remains ;)

Reply:
I don't care afound `F`, but I don't see the point for having these two constants. `ERR_PRINTER_ON_FIRE` is enough.

Reply:
Yes, keeping `ERR_PRINTER_ON_FIRE` I think is hilarious. I think the OMG one can go though as this can still come across as profanity.

Reply:
Akien graciously allowed us to get rid of both and keep ERR_PRINTER_ON_FIRE for *the greater good*.